tests/accessibility/create-test-user.ts

```diff
@@ -0,0 +1,4 @@
+import { TestUserClient } from 'nhs-notify-web-template-management-ui-tests/helpers/test-user-client';
+
+const testUserClient = new TestUserClient('.');
+testUserClient.createTestUser('accessibility-test@nhs.net', 'Test-Password1');
```

Comment:
Can we use a secure randomly generated password rather than hard coded to keep any deployed resources protected.

Reply:
Can we also use an email that identifies us a bit better (in case somebody wonders where all these test emails come from), e.g. `nhs-notify-automated-accessibility-testing@nhs.net`

Reply:
updated - also while doing this realised that there could be a problem here with running the tests multiple times in parallel so i've added a UUID at the end of the email to avoid that